src/modules/builtin/mv.rs

```diff
@@ -31,10 +31,37 @@ impl SyntaxModule<ParserMetadata> for Mv {
     fn parse(&mut self, meta: &mut ParserMetadata) -> SyntaxResult {
         syntax(meta, &mut self.modifier)?;
         self.modifier.use_modifiers(meta, |_this, meta| {
+            // Manual check for 'mv' keyword to avoid conflict with reserved keyword check
```

Comment:
I cannot see the difference between this module and other modules, so what is the manual check?